conduct.md

```diff
@@ -56,9 +56,11 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+Students are encouraged to:
+
+- Participate in class discussions.
```

Comment:
Please provide an additional sentence for each bullet point that provides an example/definition of 'participation', 'when needed', and 'regularly'. For example, is participation limited to a certain number of student input per class? Is 'when needed' up to the student to decide or for the team? Does attending regularly rely on a certain number of absences and does it apply to the lab and practical sessions?